1 - Loading in count data, normalization & QC.Rmd

```diff
@@ -0,0 +1,195 @@
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Gene_xyz_KO_1), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Gene_xyz_KO_2), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Gene_xyz_KO_3), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Gene_xyz_KO_4), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Wildtype_1), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Wildtype_2), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Wildtype_3), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+
+ggplot(RNA_seq_counts_final) +
+  xlim(-5, 100) +
+  geom_histogram(aes(x = Wildtype_4), stat = "bin", bins = 100) +
+  xlab("Raw expression counts") +
+  ylab("Number of genes")
+```
+
+
+
+Modeling count data for Tumour spheres and Adherent conditions.
+For RNA-seq with 4 treatment and 4 control, treatment = columns 1:4, treatment = control 5:8
+```{r echo=FALSE, warning=FALSE, message=FALSE}
+treatment_mean_counts <- apply(RNA_seq_counts_final[, 1:4], 1, mean) # Control means
+treatment_variance_counts <- apply(RNA_seq_counts_final[, 1:4], 1, var) # Control Variance
+treatment_variance_vs_means <- data.frame(treatment_mean_counts, treatment_variance_counts)
+
+ggplot(treatment_variance_vs_means) +
+  geom_point(aes(x=treatment_mean_counts, y=treatment_variance_counts)) + 
+  geom_line(aes(x=treatment_mean_counts, y=treatment_mean_counts, color="red")) +
+  guides(color=guide_legend(title="Treatment\nvariance = mean")) +
+  scale_y_log10() +
+  scale_x_log10()
+
+control_mean_counts <- apply(RNA_seq_counts_final[, 5:8], 1, mean) # Treatment means
+control_variance_counts <- apply(RNA_seq_counts_final[, 5:8], 1, var) # Treatment variance
+control_variance_vs_means <- data.frame(control_mean_counts, control_variance_counts)
+
+ggplot(control_variance_vs_means) +
+  geom_point(aes(x=control_mean_counts, y=control_variance_counts)) + 
+  geom_line(aes(x=control_mean_counts, y=control_mean_counts, color="red")) +
+  guides(color=guide_legend(title="Control\nvariance = mean")) +
+  scale_y_log10() +
+  scale_x_log10()
+```
+
+
+
+Filter out low read-counts before creating DESeq2Dataset object
+This is done with edgeR's cpm function.
+Rh30 low CPM filtering
```

Comment:
Change Rh30 to RNA-Seq sample?

Also maybe add in a note that this is optional, and subject to user preference - aka refer to the DESeq2 part below where the recommended practice is not to filter?

Reply:
Removed the Rh30 part and added a note for optional part.